coredns/plugin/README.md

```diff
@@ -0,0 +1,2 @@
+# kuadrant CoreDNS Plugin
+
```

Comment:
Do we expect others might want to make use of this in their own core dns? Should we add more details and consider making it available to others to use?

Reply:
I intend to add some content to this README before i move it from draft, but in terms of someone wanting to include this plugin in their own CoreDNS then it would be done in the same way as any other external plugin as described [here](https://coredns.io/2017/07/25/compile-time-enabling-or-disabling-plugins/).

Reply:
Updated. I wrote the README in a similar style to all the other COreDNS plugins